appveyor.yml

```diff
@@ -11,9 +11,9 @@ cache:
 # what combinations to test
 environment:
   matrix:
+    - nodejs_version: 'stable'
```

Comment:
Would it make sense to change this to 10 so that we are more explicit and match the rest node-10 references? Will this still work with stable once 12 is out and we still don't have a node-12 reference?